apps/yarn.lock

```diff
@@ -9635,13 +9635,25 @@ lezer-tree@^0.13.0:
   resolved "https://registry.yarnpkg.com/lezer-tree/-/lezer-tree-0.13.0.tgz#199757338a30e4c5ee8d9a767546d7ce8935bfbe"
   integrity sha512-YqGD22WxQmT6np2JEX5Z+zhdBSfPEcQ8B6ct+39oLzSAHecUPUd/maN5PKNtnUZbSzd81mEx1ZqzMDA1eRSRcg==
 
+lezer-tree@^0.13.2:
```

Comment:
First time upgrading our apps dependencies -- these look like code-mirror dependencies, but is there any best practice for confirming that I'm not accidentally bumping the version of things that I didn't intend to change?

Reply:
the only way I know to do that would be to search in this file to see if other packages depend on lezer-tree.